build_resolvers/test/resolver_test.dart

```diff
@@ -738,36 +738,37 @@ int? get x => 1;
              }
           ''',
         }, (resolver) async {
```

Comment:
Does this still have to be async? Or should the calls actually be `await resolver.libraryFor` as below?

Reply:
Actually, reverted that attempt at a fix.

I realized both these tests were missing `resolvers: createResolvers()`, which means all four test variants were testing the same shared instance. So, that's the correct fix--done. Thanks!